client/layout/guided-tours/utils.js

```diff
@@ -1,10 +1,9 @@
-/**
- * External dependencies
- */
-import { negate as not } from 'lodash';
-
 const noop = () => {};
 
 const and = ( ...conditions ) => () => conditions.every( ( cond ) => cond() );
 
+const not = function ( func ) {
+	return ! func.apply( null, arguments );
+};
```

Comment:
Is this correct? Does it translate to :

```js
const not = function ( func, ...args ) {
  return ! func.apply( null, [ func, ...args ] );
}
```

Or to

```js
const not = ( fn ) => ( ...args ) => ! fn( ...args );
```

I think the second is the correct version we want right? But I believe the current version would be the first 🤔

Reply:
I agree. I seem to have overlooked this, the second version seems more accurate and concise.